### DISCUSSION TITLE
send message without context/update

### DISCUSSION THREAD

**Discussion by yasuoiwakura** [python-telegram-bot/python-telegram-bot#2428] [Q&A]
hi there,

i'm running a bot to check the weather hourly and then post it to 1-2 channel if conditions meet (in my case: if the active Pokemon Go raidboss is boosted by the current weather then tell players)

But i have to start the bot manually using /set or something like that to
- get a context for      `context.bot.send_message(chat_id=db['chats']['TestG']['chat_id'], text=text)`
how to send messages without "context"? Just send a message to the given channel?

so how can i just send a message from a plain function without any given args?
`def msg():
 text=f"my fancy text"
 send_message(chat_id=-100123456, text=text)
 return None

updater = Updater(TOKEN, persistence=persistence)
dispatcher = updater.dispatcher
dispatcher.add_handler(CommandHandler("help",   help))

if __name__ == '__main__':
    main()`

The point is, i want to run hourly updates and THEN decide whom to send the messages.
i currently use the examples pollbot.py timerbot.py but they all use context to some user message HAS to trigger the timer to start running.

so why i just cannot just use updater.send_message() just want to use the bot and not any handler/context/update...

**Comment by Bibo-Joshi** [comment]
> so why i just cannot just use updater.send_message() just want to use the bot and not any handler/context/update...

You can (basically): [`Updater.bot`](https://python-telegram-bot.readthedocs.io/en/stable/telegram.ext.updater.html#telegram.ext.Updater.bot), [`Updater.dispatcher`](https://python-telegram-bot.readthedocs.io/en/stable/telegram.ext.updater.html#telegram.ext.Updater.dispatcher), [`Dispatcher.bot`](https://python-telegram-bot.readthedocs.io/en/stable/telegram.ext.dispatcher.html#telegram.ext.Dispatcher.bot)

You don't even need `Updater` or `Dispatcher` at all. See the [introduction](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Introduction-to-the-API) ;)
